tests/write_file/BUILD

```diff
@@ -0,0 +1,83 @@
+load("//rules:write_file.bzl", "write_file")
```

Comment:
Apologies, but I'm really at a bit of a loss for what this BUILD file is accomplishing with its various genrule -> write targets. Perhaps a top-level comment to describe the tree might help with readability?

Reply:
Done.